configs/original-dst-cluster/proxy_config.json

```diff
@@ -1,7 +1,10 @@
 {
     "listeners": [{
         "address": "tcp://0.0.0.0:10000",
-        "use_original_dst": true,
+        "listener_filters": [{
```

Comment:
Can we please make this a v2 only feature? For v1 I would recommend just internally converting "use_original_dst" into a filter internally. Otherwise this is a breaking config change.

Reply:
Happy to make this v2 only, simplifies it a little as well. It was not, however, a breaking config change, I had used this to test v1 access to the listener filters.